# Generated by Django 5.1 on 2024-08-15 01:55

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('auth', '0012_alter_user_first_name_max_length'),
    ]

    operations = [
        migrations.AlterField(
            model_name='user',
            name='username',
            field=models.CharField(error_messages={'unique': 'Ese nombre de usuario ya existe! Proba con otro'}, help_text='No mas de 150 characteres. Solo Letras, digitos y @/./+/-/_ .', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='Nombre de usuario'),
        ),
    ]
